react/AddressForm.tsx

```diff
@@ -77,7 +78,7 @@ const AddressForm: StorefrontFunctionComponent = () => {
   )
 
   const hasWithoutNumberOption = (label: string) => {
-    return label.length >= 6 && label.substr(label.length - 6) == 'Option'
+    return label.length >= 6 && label.substr(label.length - 6) === 'Option'
```

Comment:
I would prefer to use a variable here saying what this magic number means.

Reply:
sure! but I don't know what number is this :joy:

Reply:
:joy:

Reply:
But considering how many open PRs this repo has, I'm not against merge it that way for now. In the future we gonna be able to understand what this number means.

Reply:
ok! I'm trying to fix some ts errors to merge this and move on to the next ones

Reply:
> In the future we gonna be able to understand what this number means.

I should save this comment so in the future I can show you how wrong you were 🤣

Reply:
HAHAHA well, when constructing an address you have various parts like "city", "street", "state", etc., and those get mapped depending on country to "fantasy names", like "province", "stateAbbreviation", "number", "numberOption".
The thing is, there were two fantasy names that had "Option" at the end, that had a very different functionality (a checkbox gets rendered that allows to disable the number input).
Howeeeeeeeeeeeeeeever, this could easily be done with Javascript's `.endsWith()` string function 🤦‍♂.

Reply:
thanks for the heads-up Maxi! 😄